packages/order/src/services/__tests__/util/actions/exchanges.ts

```diff
@@ -96,18 +96,20 @@ describe("Order Exchange - Actions", function () {
       actions: actions,
     })
 
-    expect(changes.summary).toEqual({
-      transactionTotal: 0,
+    const sumToJSON = JSON.parse(JSON.stringify(changes.summary))
+    expect(sumToJSON).toEqual({
+      transactionTotal: "0",
```

Comment:
q: is the non stringified version a big number?

Reply:
It's because that wasn't affected by any calculation internally. So it has the type BigNumber which transforms it to numeric when toJSON is called..
When things are affected by calculations, they become a BigNumberJS, that will be a string when toJSON is called.
But these details are not relevant to the module's consumer.